components/Navigation/index.js

```diff
@@ -0,0 +1,42 @@
+import Link from "next/link";
+import { useState } from "react";
+import styled from "styled-components";
+
+const NavbarLink = styled(Link)`
+  color: black;
+  text-decoration: none;
+  padding: 0.2rem;
+  background: ${(props) => (props.active === "true" ? "grey" : "lightgrey")};
+  align-items: center;
+  font-size: 2rem;
+  transition: background 0.5s, color 0.5s;
+  &:hover {
+    color: white;
+    background: grey;
+  }
+`;
+
+export function NavBar() {
+  const [currentPage, setCurrentPage] = useState(1);
```

Comment:
i reverted this like it was before because i got errors and i dont know at the monent how to solve them and my solution was working without errors.  i mentioned this error below.